torch/profiler/profiler.py

```diff
@@ -434,14 +434,22 @@ class ProfilerAction(Enum):
 
 
 def schedule(
-    *, wait: int, warmup: int, active: int, repeat: int = 0, skip_first: int = 0
+    *,
+    wait: int,
+    warmup: int,
+    active: int,
+    repeat: int = 0,
+    skip_first: int = 0,
+    skip_first_wait: bool = False,
 ) -> Callable:
     """
     Returns a callable that can be used as profiler ``schedule`` argument. The profiler will skip
     the first ``skip_first`` steps, then wait for ``wait`` steps, then do the warmup for the next ``warmup`` steps,
     then do the active recording for the next ``active`` steps and then repeat the cycle starting with ``wait`` steps.
     The optional number of cycles is specified with the ``repeat`` parameter, the zero value means that
-    the cycles will continue until the profiling is finished.
+    the cycles will continue until the profiling is finished. The ``skip_first_wait`` parameter controls
+    whether the first wait stage should be skipped. This can be useful if a user wants to wait larger than skip_first
+    between cycles but not for the first profile.
```

Comment:
would be helpful to have some example here for the wait and skip_first occurances

Reply:
added example in newest diff